### PR TITLE
added envFrom statement to the flower deployment

### PR DESCRIPTION
**Pull request by riccardo-salamanna**
Added `envFrom` statement to the flower deployement in the helm chart
---
The flower deployement is missing the envFrom statement. This causes that if you use the `extraEnvFrom` in the value files,  they are not deployed on all the pods.

### CODE REVIEW

File: chart/templates/flower/flower-deployment.yaml [hunk 1]
```diff
@@ -128,6 +128,8 @@ spec:
                   name: {{ template "flower_secret" . }}
                   key: basicAuth
             {{- end }}
+          envFrom:
+          {{- include "custom_airflow_environment_from" . | default "\n  []" | indent 10 }}
```
Comment (jedcunningham) [inline chart/templates/flower/flower-deployment.yaml]: This can't go in between stuff in `env`. Either move it above or below the whole section.

**Review by jedcunningham — CHANGES_REQUESTED** [review]
Can you also add flower to the [chart/tests/test_extra_env_env_from.py](https://github.com/apache/airflow/blob/main/chart/tests/test_extra_env_env_from.py) tests?

File: chart/templates/flower/flower-deployment.yaml (hunk 1 above)
Comment (riccardo-salamanna) [reply chart/templates/flower/flower-deployment.yaml]: Thanks for answering and pointing those things out. i **think** it should be better now.

**Review by jedcunningham — APPROVED** [review]

**Comment by riccardo-salamanna** [comment]
made a mistake with the commits... i was trying to open a different PR and forgot this was open

**Review by jedcunningham — APPROVED** [review]

**Comment by jedcunningham** [comment]
@riccardo-salamanna, the helm unit tests are failing. Can you take a look at that?

**Comment by riccardo-salamanna** [comment]
> @riccardo-salamanna, the helm unit tests are failing. Can you take a look at that?

sure, i'll do my best  - p.s. i noted also that the triggerer deployment has the envFrom statement but it is outside of the testing

**Review by jedcunningham — APPROVED** [review]

**Comment by jedcunningham** [comment]
Thanks @riccardo-salamanna! Congrats on your first commit 🎉

**Pull request merged**